src/index.js

```diff
@@ -43,6 +44,24 @@ $('#js-solve-button').on('click', function() {
 
 $('#js-submit-button').on('click', function(e) {
   e.preventDefault();
-  domUpdates.checkUserGuess(game);
+  let letterInput = domUpdates.grabCurrentLetter();
+  game.currentPuzzle.checkUserGuess(letterInput);
+  $('.input').val('');
+});
+
+$('#js-vowel-submit').on('keyup', function(e) {
+  e.preventDefault();
+  domUpdates.checkVowel();
+});
+
+$('#js-vowel-submit-button').on('click', function(){
+  let letterInput = domUpdates.grabVowel();
+  game.currentPuzzle.checkUserGuess();
```

Comment:
Are you supposed to pass letterInput to checkUserGuess()?
If not where is letterInput going?